typings/generated/src/Device/deviceSetup.d.ts

```diff
@@ -1,28 +1,50 @@
 /// <reference types="node" />
-import { Device, TDispatch } from '../state';
+import { Device, RootState, TDispatch } from '../state';
 import { InitPacket } from './initPacket';
-import { PromiseChoice, PromiseConfirm } from './sdfuOperations';
 export interface DfuEntry {
+    key: string;
+    description?: string;
     application: string;
     semver: string;
     softdevice?: string | Buffer;
     params: Partial<InitPacket>;
 }
+export interface JprogEntry {
+    key: string;
+    description?: string;
+    fw: string;
+    fwIdAddress: number;
+    fwVersion: string;
+}
+type PromiseChoice = (question: string, choices: string[]) => Promise<{
+    choice: string;
+    index: number;
+}>;
+export type PromiseConfirm = (message: string) => Promise<boolean>;
+export interface IDeviceSetup {
```

Comment:
Rename `IDeviceSetup`.

We do not use `I` as a prefix for interfaces. Here it is even worse because there is `DeviceSetup` and `IDeviceSetup`.

Reply:
Any suggestion on this @datenreisender not sure what to name it.